views/userDetails.njk

```diff
@@ -96,21 +96,40 @@
                   </td>
                 {% endif %}
               {% else %}
-                <td class="govuk-table__cell">
+                {% if staff.accountStatus == 'EXPIRED' %}
+                  <td class="govuk-table__cell">
+                  {{ govukTag({
+                    text: 'Inactive account - ' + staff.accountStatus,
+                    classes: "govuk-tag--yellow",
+                    attributes: { "data-qa": "enabled" }
+                  }) }}
+                  </td>
+                  {% if showEnableDisable %}
+                    <td class="govuk-table__cell govuk-table__cell--numeric govuk-!-width-one-quarter">
+                      {{ linkButton(csrfToken, {
+                        text: "Deactivate account",
+                        action: staffUrl + "/deactivate",
+                        id: "deactivate-button"
+                      })}}
+                    </td>
```

Comment:
Expired now displayed in yellow and if have permissions, can deactivate/lock the user (An 'EXPIRED' user is ALWAYS unlocked)